CRVReco/src/CrvPedestalFinder_module.cc

```diff
@@ -35,53 +39,121 @@ namespace mu2e
       using Comment=fhicl::Comment;
       fhicl::Atom<std::string> crvDigiModuleLabel{Name("crvDigiModuleLabel"), Comment("module label for CrvDigis")};
       fhicl::Atom<double>      maxADCspread{Name("maxADCspread"), Comment("maximum spread of ADC values within a waveform to be considered for the pedestal")};
+      fhicl::Atom<std::string> tmpDBfileName{Name("tmpDBfileName"), Comment("name of the tmp. DB file name for the pedestals")};
```

Comment:
I'm curious why this parameter has to be entered for this module: can't you specify a text override for the relevant condition via the database interface?

Reply:
This module generates a temporary DB text file with the pedestals it finds. These pedestals are used by the calibration module in the next step (via this DB text file, i.e. using the text override). Once it has been used there, it can be discarded.
There is a similar mechanism for the calibration module, where also a temporary DB text file will be produced. This file will be sent through the verification step, before it can be submitted to the official DB.